recipes-ni/niconfigpersistentlogs/files/niconfigpersistentlogs

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -uo pipefail
+
+error_and_die() {
+set +u
+    if [ -n "$1" ]; then
+        echo >&2 "$1"
+    fi
+set -u
+    echo >&2 "ERROR"
+    exit 1
+}
+
+FILE_PATH_VOLATILES_CONFIG="/etc/default/volatiles/00_core"
```

Comment:
I can't recall which package owns this file, but `ni-configpersistentlogs` should have an `RDEPENDS` on it.